scripts/demoScripts/create-demo-data.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo 'ORANGE_DEMO_PATIENTS is:';
+echo $ORANGE_DEMO_PATIENTS;
+
+echo 'looping...';
+for i in `seq 0 10`; do
+  PATIENT_EMAIL=$(echo $ORANGE_DEMO_PATIENTS | jq ".[$i].email")
```

Comment:
Moderate prio: Structure of `ORANGE_DEMO_PATIENTS` doesn't appear in repo.

Please add sample / example. Could kill two birds with one stone and add a test.

**Without an example/sample `ORANGE_DEMO_PATIENTS`, cumbersome to test**
(have to look at the jq and write an array as a single-line ENV)

```
$ grep 'ORANGE_DEMO_PATIENTS' * -R
scripts/demoScripts/create-demo-data.sh:echo 'ORANGE_DEMO_PATIENTS is:';
scripts/demoScripts/create-demo-data.sh:echo $ORANGE_DEMO_PATIENTS;
scripts/demoScripts/create-demo-data.sh:  PATIENT_EMAIL=$(echo $ORANGE_DEMO_PATIENTS | jq ".[$i].email")
scripts/demoScripts/create-demo-data.sh:  PATIENT_PASSWORD=$(echo $ORANGE_DEMO_PATIENTS | jq ".[$i].password")
```

Give an example to run with `npm test`, or at least write example execution in `./scripts/demoScripts/README.md`

Reply:
For the other ENVs, e.g. `$AUTH_MICROSERVICE_URL`:
- Option: Mention `source orange-api/.env.example` as quickly get the other ENVs.
- Option: Include a `.env.example` in `./scripts/demoScripts`
- Option: Hard-code `environment` in docker-compose.yml